apps/src/export.js

```diff
@@ -0,0 +1,138 @@
+import {SnackSession} from '@code-dot-org/snack-sdk';
```

Comment:
I wonder if this file should live in `apps/src/lib/util` instead of the source-root.  Just thinking for someone trying to understand our code, this isn't one of the first things we'd want them to read.

Reply:
yes, I debated that as well. we have `src/`, `src/util/`, `src/lib/util/` as potential candidates. I tried to match the convention being used (common code seemed to be at root, re-usable "utilities" seem to be in `util`) - but I can definitely see the argument against using the root of `src`

Reply:
I still try to stick to the organization proposed in https://github.com/code-dot-org/code-dot-org/blob/8990f16a29d8a1cf9dba0ffb220bd424e8542810/apps/docs/refactor.md although I'm not sure that's a shared goal anymore.

Reply:
after merging with `staging`, I've moved the code into a new file recently created by @davidsbailey called `src/util/exporter.js`